app/scripts/views/components/dataTable/tableView.js

```diff
@@ -298,7 +303,7 @@
           var freq = 0;
           datum.gene = item.gene;
           if (_selectedGenesMap === undefined) {
-            datum.case_uids = iViz.util.unique(item.case_uids);
+            datum.case_uids = item.case_uids;
```

Comment:
Are you sure `item.case_uids` be unique?

Reply:
yes, uids will be unique cases